integration/nwo/fabricconfig/orderer.go

```diff
@@ -121,3 +122,8 @@ type OrdererStatsd struct {
 	WriteInterval time.Duration `yaml:"WriteInterval,omitempty"`
 	Prefix        string        `yaml:"Prefix,omitempty"`
 }
+
+type ChannelParticipation struct {
+	Enabled       bool `yaml:"Enabled"`
+	RemoveStorage bool `yaml:"RemoveStorage"`
+}
```

Comment:
`RemoveStorage` is probably redundant right now, there is a (significant) change we won't support archiving, so probably better to leave it out for now.